cmd/procs/description/descriptor.go

```diff
@@ -18,6 +18,11 @@ func NewCmd(printer io.Printer, proctorEngineClient daemon.Client) *cobra.Comman
 		Run: func(cmd *cobra.Command, args []string) {
 			procList, err := proctorEngineClient.ListProcs()
 			if err != nil {
+				if err.Error() == "Unauthorized" {
```

Comment:
Please extract "Unauthorized" as a utility error/constant because it's used in multiple places.

Then you can do a direct check

if err == utils.UnauthorizedError instead of if err.Error() == "Unauthorized"